scripts/doctests.py

```diff
@@ -36,6 +50,24 @@ def check_rstorm(dirn):
                 else:
                     print(f'Ran {ofile} successfully.')
 
+def check_changelogs(dirn):
+    # Ensure all changelog files are valid yaml and do not contain multiline scalars
+    for fn in os.listdir(dirn):
+        if not fn.endswith('.yaml'):
+            continue
+        fp = os.path.abspath(os.path.join(dirn, fn))
+        print(f'Checking {fp}')
+        with open(fp, 'rb') as fd:
+            bytz = fd.read()
+        # Do we have multi-line scalers?
+        if re.findall(r'[a-z0-9]\:\s+\|\s*\n', bytz.decode('utf8'), flags=re.IGNORECASE):
+            raise ValueError(f'multiline scaler detected in {fp}')
+        # Just asserting we are valid yaml to start with.
+        data = yaml.load(bytz, yaml.CSafeLoader)
```

Comment:
CSafeLoader isn't guaranteed to be present. Maybe `s_common.Loader` instead (which does the fallback dynamically)?

Reply:
Just switched to SafeLoader.